radicle-cob/src/history.rs

```diff
@@ -16,14 +16,17 @@ use crate::pruning_fold;
 pub mod entry;
 pub use entry::{Contents, Entry, EntryId};
 
-#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash, serde::Serialize, serde::Deserialize)]
+#[derive(
+    Clone, Copy, Debug, Default, PartialEq, Eq, Hash, serde::Serialize, serde::Deserialize,
+)]
+#[serde(rename_all = "lowercase")]
 pub enum HistoryType {
+    #[default]
+    Default,
```

Comment:
I might have left a similar comment somewhere else, but I would suggest this be named `Heartwood` or `Radicle`.

Reply:
I think we should remove the type entirely as it doesn't really make sense; it would be best if `radicle-cob` wasn't tied to a fixed set of CRDTs.